soap/index.js

```diff
@@ -101,6 +101,8 @@ module.exports = yeoman.Base.extend({
         }
       }
       self.url = self.selectedDS.data.wsdl;
+      self.log(chalk.green(g.f('WSDL for %s datasource: %s',
```

Comment:
@raymondfeng I had it like you suggested before, but after reading printed sentence, I changed it.. anyway, I am fine with either way.. changed it to your suggestion.